.github/workflows/kani.yml

```diff
@@ -6,8 +6,8 @@ on:
   push:
     # Not just any push, as that includes tags.
     # We don't want to re-trigger this workflow when tagging an existing commit.
-    branches:
-      - main
```

Comment:
I think `branches: *` might also have worked, but as long as this still ignores the tags successfully, lgtm.

Reply:
indeed.

Reply:
Actually, I broke everything. :( I'm implementing your solution instead.